Add Template to Core Guide to Kusama (#5879)
w3f/polkadot-wiki: docs/build/build-guides-coretime-start.md

[core](../learn/learn-agile-coretime.md#core), which is how one utilizes the shared security of the
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} network. One does so by:

```mermaid
%%{
 init: {
 'theme': 'base',
 'themeVariables': {
 'fontFamily': 'Unbounded',
 'primaryColor': '#E6007A',
 'fontSize': '16px',
 'primaryTextColor': '#fff',
 'primaryBorderColor': '#7C0000',
 'lineColor': '#140523',
 'secondaryColor': '#552BBF',
 'tertiaryColor': '#fff'
 }
 }
}%%
flowchart TD
    subgraph GA["Generate Artifacts"]
        direction LR
        A["Creating a runtime"]-->B["Compiling to Wasm"]-->C["Generate Genesis State"]
    end

    subgraph PC["Procure ParaId & Core"]
        direction LR
        PARAID["Reserve ParaId"]
        PARAID-->D["Buy Bulk Coretime"]
        PARAID-->E["Issue On-Demand Coretime Extrinsic"]
    end

    subgraph DEP["Deploying"]
        direction LR
        F["Register artifacts to ParaId"]-->assign["Assign Core"]-->G["Sync collator"]-->H["Begin generating blocks!"]
    end

GA-->PC
PC-->DEP
```

### Install dependencies

Make sure you have everything you need for your target system
[here.](./build-guides-install-deps.md)

> ### Deploy adder collator

Reviewer: ```suggestion
### Deployment Example - Adder Collator
```

Try out the above by deploying the
[adder collator, a very simple "counter" parachain implementation.](../learn/learn-guides-coretime-parachains.md).

> ## OpenZepplin Templates & Guides

Reviewer: ```suggestion
## OpenZeppelin Templates & Guides
```

> OpenZepplin offers a generic parachain template, which has support for:

Reviewer: ```suggestion
OpenZeppelin offers a [generic parachain template](https://github.com/OpenZeppelin/polkadot-generic-runtime-template), which has support for:
```

- Proxy Pallet
- Multisig Pallet
- Governance support - a treasury, referenda (OpenGov!), and assets configuration
- Collation/Parachain Support
- XCM (Cross Consensus Messaging) Configuration and Support

You can find more about it in their

> [Substrate parachain runtime guide.](https://docs.openzeppelin.com/substrate-runtimes/1.0.0/)

Reviewer: ```suggestion
For more information, check their
[Substrate parachain runtime guide.](https://docs.openzeppelin.com/substrate-runtimes/1.0.0/)
```

> ## Template to Core

Reviewer: ```suggestion
## Polkadot SDK Parachain Template
```

If you wish to use a template, such as the ones found in the Polkadot SDK, feel free to use the

> [Template to Core guide.](./build-guides-template-basic.md)

Reviewer: ```suggestion
If you wish to the [Polkadot SDK's Parachain template](https://github.com/paritytech/polkadot-sdk/tree/master/templates/parachain), please follow the
[Template to Core guide.](./build-guides-template-basic.md)
```
